pkg/api/validation/validation_test.go

```diff
@@ -2881,6 +2924,121 @@ func TestAlphaHugePagesIsolation(t *testing.T) {
 	}
 }
 
+func TestAlphaVolumeMode(t *testing.T){
+	validMode := api.PersistentVolumeFilesystem
+	invalidMode := api.PersistentVolumeMode("fake")
+	validName := "valid"
+
+	// PersistentVolume with VolumeMode set (valid and invalid)
+	validModePV := api.PersistentVolume{
```

Comment:
I would write a wrapper to generate the PV and PVC objects, taking in mode as a parameter.  I would test the valid modes "filesystem" and "block", and also the invalid modes, and nil mode.